drivers/watchdog/wdt_gecko.c

```diff
@@ -259,6 +254,13 @@ static int wdt_gecko_init(const struct device *dev)
 	CMU_ClockEnable(config->clock, true);
 #else
 	CMU_ClockSelectSet(config->clock, cmuSelect_ULFRCO);
+	/* Enable Watchdog clock. */
+	CMU_ClockEnable(cmuClock_WDOG0, true);
```

Comment:
Why not use the `CMU_ClockEnable(config->clock, true);` a few lines above?

Reply:
yup, updated the code